MessengerOnFileForLinux/MessageFramework/src/Message.cpp

```diff
@@ -53,6 +53,11 @@ bool Message::setMessageFlag(std::string messageFlag)
         log_.info("Message::setMessageFlag() message read, can not unread");
         return false;
     }
+    else
+    {
+        log_.info("Message::setMessageFlag() messageFlag unknown, error");
```

Comment:
nie mieliśmy konwencji Message::setMessageFlag() ERROR: messageFlag unknown ?

Reply:
możliwe, że mieliśmy (nie kojarzę nadal), ale i tak będzie wypadała, bo w nowym Loggerze będzie to jako parametr STATUS. INFO, DEBUG, ERROR itd.